roles/os_hardening/vars/main.yml

```diff
@@ -110,3 +110,20 @@ os_security_suid_sgid_system_whitelist:
 
 # system accounts that do not get their login disabled and pasword changed
 os_always_ignore_users: ['root', 'sync', 'shutdown', 'halt']
+# Define /tmp permissions
```

Comment:
These are duplicated in `defaults` and should be removed here. Else we can't override it easily.

Reply:
removed last lines from vars/main. Is it fixed? @rndmh3ro

Reply:
Yes, thanks!